Empire/Empire.cpp

```diff
@@ -1387,6 +1387,18 @@ void Empire::AddBuildingType(const std::string& name) {
         return;
     m_available_building_types.insert(name);
     AddSitRepEntry(CreateBuildingTypeUnlockedSitRep(name));
+
```

Comment:
I don't think this sort of intro / guide sitrep generation belongs here. I'd do this in a FOCS script, possibly with a first-turn-only condition, and preferably with a test based on a rule about whether to show tutorial sitreps.

Reply:
See the comment above for how to design the required FOCS scripting.

Regarding the test whether to show tutorial sitreps: You have already designed and implemented a totally adequate and very comfortable solution for that: in the "SitRep" window, click the "Filters" button and deselect "Tutorial Hints".

Reply:
The extra rule would be to avoid generating those extra sitreps at all, to avoid having to hide them, but the existing UI feature might be sufficient (I don't use it enough to be sure).

Reply:
I see the point.  For experienced players, there is no point in turning off beginner features in each individual game, being able to turn them off in the config file does seem more convenient.  If that is implemented, the new option should probably turn off the random hints *and* the tutorial hints, and in that case, i suggest doing it in a separate commit/PR.